index.html

```diff
@@ -4452,11 +4453,13 @@ <h3>Parsing content</h3>
       </p>
       <p>
         In some cases it might be obvious that the tag/device contains privacy
-        sensitive data, say in case it is a glucose meter which can indicate
-        that you, or someone in the close family, are a diabetes patient.
+        sensitive data, say in case of NFC equipped conference badgets and business
+        cards. This would also be the case with an NFC equipped glucose meter,
+        which can indicate that you, or someone in the close family, are a diabetes
```

Comment:
s/you/the user ?

Reply:
AFAIK "you" wording is to be avoided :)

Reply:
Feel free to fix :) its surely in other places as well